dcc-portal-ui/app/scripts/ui/js/directives.js

```diff
@@ -366,9 +366,9 @@ angular.module('icgc.ui.popover', [])
 angular.module('icgc.ui.copyPaste', [])
   .provider('copyPaste', function () {
     var _provider = this,
+        //_zeroClipPath = require('../assets/ZeroClipboard.swf'), <- for local development only
         _zeroClipPath = '//cdnjs.cloudflare.com/ajax/libs/zeroclipboard/2.2.0/ZeroClipboard.swf',
```

Comment:
So, are we using a local copy or not. I thought we had decided this was the best course. I see changes to the build to reflect that, so not including it here seems inconsistent. @aishmitk please work with @cheapsteak if you are unsure how to proceed. Thanks.